examples/gopherbin-config.toml

```diff
@@ -0,0 +1,20 @@
+[default]
```

Comment:
An example config already exists here:

https://github.com/gabriel-samfira/gopherbin/blob/master/testdata/config.toml

It even includes a commented out ```TLS``` example. I think you can just use that one.

Reply:
removed examples directory